src/store/general/reducers.test.ts

```diff
@@ -0,0 +1,160 @@
+import { actions, reducer } from "./slice";
+
+const defaultState = {
```

Comment:
These tests will be updated to use factory functions in a followup (https://warthogs.atlassian.net/browse/WD-822).